src/main/java/github/weichware10/util/data/DataPoint.java

```diff
@@ -60,16 +66,34 @@ public DataPoint(int dataId, int timeOffset, int[] coordinates, int[] rasterSize
      *
      * @since v0.3
      */
-    public DataPoint(int dataId,
-            int timeOffset, int[] coordinates, int[] rasterSize,
-            Float zoomLevel) {
+    public DataPoint(int dataId, int timeOffset, int[] coordinates, int[] rasterSize) {
         this.dataId = dataId;
         this.timeOffset = timeOffset;
         this.coordinates = coordinates;
         this.rasterSize = rasterSize;
-        this.zoomLevel = zoomLevel;
+        this.zoomLevel = null;
     }
 
+    // /**
+    //  * Stores a single DataPoint with everyting (for database).
+    //  *
+    //  * @param dataId      - the id of the dataPoint
+    //  * @param timeOffset  - the time since the trial started
+    //  * @param coordinates - the coordinates on the viewed picture
+    //  * @param rasterSize  - width and height of the raster
+    //  *
+    //  * @since v0.3
+    //  */
+    // public DataPoint(int dataId,
+    //         int timeOffset, int[] coordinates, int[] rasterSize,
+    //         Float zoomLevel) {
+    //     this.dataId = dataId;
+    //     this.timeOffset = timeOffset;
+    //     this.coordinates = coordinates;
+    //     this.rasterSize = rasterSize;
+    //     this.zoomLevel = zoomLevel;
+    // }
```

Comment:
Das ist noch in Kommentaren, weil wir die Datenbankfunktionen noch nicht fertig auf Git haben ?

Reply:
Das habe ich erstmal auskommentiert, da sonst der Konstruktor doppelt gewesen wäre

Reply:
Kann ich aber auch löschen, da wir es nicht mehr benötigen